conceptual/EFCore.PG/release-notes/6.0.md

```diff
@@ -0,0 +1,22 @@
+# 6.0 Release Notes
```

Comment:
I think this page made it in inadvertently... Note that I force-pushed an amended commit for this change, so you may need to reset (nobody ever contributes to the docs except for me!)